addons/web/static/src/xml/web_calendar.xml

```diff
@@ -52,14 +52,15 @@
 
     <t t-name="CalendarView.sidebar.filter">
         <t t-set="needCollapse" t-value="widget.filters.length > 2 and widget.title"></t>
+        <t t-set="collapseTitle" t-value="widget.title and widget.title.split(' ').join('')"></t>
```

Comment:
```suggestion
        <t t-set="collapseTitle" t-value="widget.title and widget.title.replace(/ /g, '')"></t>
```

Reply:
@kba-odoo, We can not use the regular expression `/ /g` inside the qweb template. it will give error while template rendering.
Thanks